Source/Diagnostics/WarpXIO.cpp

```diff
@@ -211,8 +211,11 @@ WarpX::InitFromCheckpoint ()
         }
 
         mypc->ReadHeader(is);
-        is >> current_injection_position;
-        GotoNextLine(is);
+        for (auto& inj_pos : current_injection_position)
```

Comment:
```suggestion
        for (auto const& inj_pos : current_injection_position)
```

Reply:
This `const` does not compile, I think because we modify `inj_pos` with `is >> inj_pos` in the `for` loop.

Reply:
Yes, not this one.